### PR TITLE
OSDK-2345: Export scorecard storage as an env variable in a test pod

### PR DESCRIPTION
**Pull request by apodhrad**
Signed-off-by: Andrej Podhradsky <apodhrad@redhat.com>

**Description of the change:**

**Motivation for the change:**

**Checklist**

If the pull request includes user-facing changes, extra documentation is required:
- [ ] Add a new changelog fragment in `changelog/fragments` (see [`changelog/fragments/00-template.yaml`](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))
- [ ] Add or update relevant sections of the docs website in [`website/content/en/docs`](https://github.com/operator-framework/operator-sdk/tree/master/website/content/en/docs)

### CODE REVIEW

File: internal/scorecard/storage.go [hunk 1]
```diff
@@ -177,6 +177,13 @@ func addStorageToPod(podDef *v1.Pod, mountPath string, storageImage string) {
 	}
 	podDef.Spec.Containers[0].VolumeMounts = append(podDef.Spec.Containers[0].VolumeMounts, vMount)
 
+        // add mountPath to Env
+        mountPathEnv := v1.EnvVar{
```
Comment (jmrodri) [inline internal/scorecard/storage.go:181]: this fails `go fmt`. Should be a TAB.
```suggestion
     mountPathEnv := v1.EnvVar{
```

**Review by jmrodri — CHANGES_REQUESTED** [review]
Minor formatting issue. Other than that I think this is a good idea.

**Comment by jmrodri** [comment]
@apodhrad sadly you forgot to sign the latest commit. It is failing DCO check. You will have to do a rebase with `--signoff` to fix it, something like this `git rebase HEAD~2 --signoff`

**Review by jmrodri — APPROVED** [review]

**Comment by jmrodri** [comment]
@apodhrad actually this PR warrants a changelog entry. https://github.com/operator-framework/operator-sdk/blob/master/changelog/fragments/00-template.yaml explaining this addition.

**Review by jmrodri — APPROVED** [review]
/lgtm

**Review by theishshah — APPROVED** [review]
Looks good, shouldn't break existing storage functionality

**Pull request merged**